### PR TITLE
Update helpdesk_ticket_task.md

### PR DESCRIPTION
**Pull request by Armin83**
Added 9.1.1 task template description

### CODE REVIEW

File: en/helpdesk_ticket_task.md [hunk 1]
```diff
@@ -38,5 +38,8 @@ defining a time interval for the reminder. Notifications must be enabled
 (see [Configure
 Notifications](config_notification.html "Notifications are configured from the menu Setup > Notifications ;")).
 
+Note: For recurrent task it is possible to add a task template. You can predefine the values contenct, category and duration.
```
Comment (artmoni) [inline en/helpdesk_ticket_task.md]: contenct => content

File: en/helpdesk_ticket_task.md [hunk 2]
```diff
@@ -38,5 +38,8 @@ defining a time interval for the reminder. Notifications must be enabled
 (see [Configure
 Notifications](config_notification.html "Notifications are configured from the menu Setup > Notifications ;")).
 
+Note: For recurrent task it is possible to add a task template. You can predefine the values contenct, category and duration. 
+Hint: You should always selcect first a template before change something in text field. The selection will override your text.
```
Comment (artmoni) [inline en/helpdesk_ticket_task.md]: selcect => select

**Comment by Armin83** [comment]
@trasher where is the english documentation gone ? https://github.com/Armin83/doc/blob/master/en/helpdesk_ticket_task.md

**Comment by trasher** [comment]
New documentation is French only, I've posted months ago on the doc mailing list to explain.

**Comment by Armin83** [comment]
Thanks for response - so i can do nothing to constribute at the moment.
https://mail.ow2.org/wws/arc/glpi-doc/2017-11/msg00000.html

**Comment by trasher** [comment]
Indeed, until french doc has not been finished (or until someone fully update/rewrite the english version)... That's also why we've splitted install doc.

**Pull request closed without merging**